src/main/kotlin/org/usecase/user/process/HealthCheckProcess.kt

```diff
@@ -0,0 +1,47 @@
+package org.usecase.user.process
```

Comment:
The same idea above. I think it is better to place all these files (HealthCheckRouter, HealthCheckProcess, HealthCheckDao) inside `org.usecase.user.app.healthCheck`.